### PR TITLE
Update ui tests to run cleanly again

### PR DESCRIPTION
**Pull request by ldimaggi**
[16:23:00] I/launcher - 0 instance(s) of WebDriver still running
[16:23:00] I/launcher - chrome #01-0 passed
[16:23:00] I/launcher - chrome #01-1 passed
[16:23:00] I/launcher - chrome #01-2 passed
[16:23:00] I/launcher - chrome #01-3 passed
[16:23:00] I/launcher - chrome #01-4 passed
[16:23:00] I/launcher - chrome #01-5 passed
[16:23:00] I/launcher - chrome #01-6 passed
[16:23:00] I/launcher - chrome #01-7 passed
[16:23:00] I/launcher - chrome #01-8 passed
[16:23:00] I/launcher - chrome #01-9 passed
[16:23:00] I/launcher - chrome #01-10 passed
[16:23:00] I/launcher - chrome #01-11 passed
[16:23:00] I/launcher - chrome #01-12 passed
[16:23:00] I/launcher - chrome #01-13 passed
[16:23:00] I/launcher - chrome #01-14 passed
[16:23:00] I/launcher - chrome #01-15 passed
[16:23:00] I/launcher - chrome #01-16 passed
[16:23:00] I/launcher - chrome #01-17 passed
[16:23:00] I/launcher - chrome #01-18 passed

### CODE REVIEW

**Comment by fabric8cd** [comment]
@ldimaggi fabric8/fabric8-ui:SNAPSHOT-PR-2305-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2305-fabric8-planner.badger.fabric8.io

**Comment by rgarg1** [comment]
@ldimaggi Are you sure the v2.33 of webdriver works with older versions of chrome?

File: runtime/tests/protractor.config.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ exports.config = {
     suites: {
       smokeTest: './../src/tests/**/smokeTest.spec.js',
       fullTest:  './../src/tests/**/*.spec.js',
-      tempTest: './../src/tests/**/iteration.spec.js'
+      tempTest: './../src/tests/**/*quickadd-workitem*.spec.js'
```
Comment (jarifibrahim) [inline runtime/tests/protractor.config.js]: @ldimaggi Do we need this?

**Comment by jarifibrahim** [comment]
I ran the PR locally and all the tests are passing :+1:

**Comment by fabric8cd** [comment]
@ldimaggi fabric8/fabric8-ui:SNAPSHOT-PR-2305-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2305-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@ldimaggi fabric8/fabric8-ui:SNAPSHOT-PR-2305-3 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2305-fabric8-planner.badger.fabric8.io

File: runtime/src/tests/work-item/work-item-list/displayAndUpdateWorkItemDetails.spec.js [hunk 2]
```diff
@@ -347,25 +347,6 @@ it('Verify how many work item type exists in drop down - desktop', function() {
   });
 
   /*Verfify on selecting workitem it should display in list and detail view both pages */
-  it('Verfify on selecting workitem state it should display in list and detail view both pages -phone ', function() {
```
Comment (jarifibrahim) [inline runtime/src/tests/work-item/work-item-list/displayAndUpdateWorkItemDetails.spec.js]: @ldimaggi Why delete this test?

Comment (ldimaggi) [reply runtime/src/tests/work-item/work-item-list/displayAndUpdateWorkItemDetails.spec.js]: This test - and similar tests should be removed and perhaps recreated. What we want to create as a replacement is a test to verify dynamic changes. Also - the test as written hard codes a reference to an arry element with a string used to define an icon. This is a very brittle approach.

Once the current spring ends - we should review ALL the existing tests and decide which to retain, which to remove, which to update, and which to totally recreate. Thx!

Comment (jarifibrahim) [reply runtime/src/tests/work-item/work-item-list/displayAndUpdateWorkItemDetails.spec.js]: Right :+1:

**Comment by fabric8cd** [comment]
@ldimaggi fabric8/fabric8-ui:SNAPSHOT-PR-2305-4 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2305-fabric8-planner.badger.fabric8.io

**Pull request merged**